qiita_pet/util.py

```diff
@@ -51,8 +56,13 @@ def clean_str(item):
     -------
     str
         cleaned string
+
+    Notes
+    -----
+    This function removes colons and periods from strings, and replaces spaces
+    with underscores.
     """
-    return str(item).replace(" ", "_").replace(":", "")
+    return re.sub(clean_str_regex, "", str(item).replace(" ", "_"))
```

Comment:
Unfortunately no, since the `.` and `:` are being replaced with something different than the space (empty string versus underscore, respectively). It could be implemented like this:

```python
clean_str_regex = re.compile("[.:]")
def repl_fn(matchobj):
    if matchobj.group(0) == ' ':
        return '_'
    else:
        return ''

def clean_str(item):
    return re.sub(clean_str_regex, repl_fn, str(item))
```

but I'm not sure if that would be faster (it's calling a function for each occurrence of any of the three characters specified).